scripts/src/main/resources/scripts/command/aws

```diff
@@ -51,21 +64,19 @@ function doSetup() {
       AWS_VERSION=$(curl "https://github.com/aws/aws-cli/tags" | awk -F'tags/' '/archive/ { print $2}' | sort -r | head -1 | awk -F'.zip' '{print $1}')
     fi
 
-    doInstall "-" "${AWS_HOME}" "aws" "${AWS_VERSION}" 
```

Comment:
wouldn't it be better if we keep the **AWS_HOME** variable and use it here as before? it was set to the same given value here.

Reply:
Using AWS_HOME does not work, because it means, we install the software on all systems in the same directory, and that is not possible.